.changeset/serious-rivers-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/webchannel-wrapper": patch
+---
+
+fix: Update webchannel-wrapper to fix webchannel multi-byte character decoding bug.
```

Comment:
Thoughts on "Update webchannel-wrapper to fix webchannel multi-byte character decoding bug in fetch streams."

Reply:
Good idea. Done.